### PR TITLE
Updating README.md for Advanced Console UIs …

### PR DESCRIPTION
**Pull request by developer-guy**
I added another project related to "Advanced Console UIs" .

Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo:
- godoc.org:
- goreportcard.com:
- coverage service link ([gocover](http://gocover.io/), [coveralls](https://coveralls.io/) etc.)

Very good coverage

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added godoc link to the repo and to my pull request.
- [x] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:

### CODE REVIEW

**Comment by ceriath** [comment]
Please fill in the PR template

File: README.md [hunk 1]
```diff
@@ -242,6 +242,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [go-colortext](https://github.com/daviddengcn/go-colortext) - Go library for color output in terminals.
 * [go-isatty](https://github.com/mattn/go-isatty) - isatty for golang.
 * [go-prompt](https://github.com/c-bata/go-prompt) - Library for building a powerful interactive prompt, inspired by [python-prompt-toolkit](https://github.com/jonathanslenders/python-prompt-toolkit).
+* [promptui](https://github.com/manifoldco/promptui) Interactive prompt for command-line applications https://www.manifold.co
```
Comment (kirillDanshin) [inline README.md]: please remove the external link. also please fix your item line to be in form of
```
* [name](https://github.com/username/repo) – Description.
```

and put the link and alphabetical order.

Comment (developer-guy) [reply README.md]: @kirillDanshin ok i fixed it.

**Comment by developer-guy** [comment]
@kirillDanshin come on man , apply this merge request :)

**Comment by ceriath** [comment]
Please fill in the PR template

**Comment by kirillDanshin** [comment]
@developer-guy we can't merge this PR until you fill in the PR template and go thru our checklist

**Pull request closed without merging**